examples/golang/docker-compose-go.yaml

```diff
@@ -2,10 +2,13 @@ services:
   go_plugin_compile:
     build:
       context: ../shared/build
```

Comment:
should we just use a golang container in this case ?

there are some recipes already using it check `shared/golang` and any examples that link to it